recOrder/cli/compute_transfer_function.py

```diff
@@ -13,8 +13,19 @@
 
 
 def compute_transfer_function_cli(config_path, output_path):
+    """CLI command to compute the transfer function given a configuration file path 
+    and a desired output path. Given no arguments, a default configuration file will 
+    be used and the output will be in transfer-function.zarr.
+
+    Parameters
+    ----------
+    config_path : string
+        Path of the configuration file.
+    output_path : string
+        Path of the output file.
+    """
     # Load config file
-    if config_path is None:
+    if config_path is None or "None":
```

Comment:
When will this be a literal `"None"`?

Reply:
I believe Talon wanted the function to take in a literal "None" just in case.

Reply:
I don't think this case should be allowed. If a user wants to trigger the default settings explicitly we can add a `"default"` option.

Reply:
I don't think this case should be allowed. If a user wants to trigger the default settings explicitly we can add a `"default"` option.

Reply:
Sorry I made a misleading suggestion. I agree that we should drop the `or "None":` here. 

My (very minor) concern that I mentioned in person to Clinton was that the "recreation strings" can be incorrect if you don't provide an argument. 

So `recorder compute-transfer-function` would print:
```
Recreate this transfer function with:
>> recorder compute-transfer-function None -o ./transfer-function.zarr
```
which won't actually run. 

I can quickly fix this.